examples/P5C060.toml

```diff
@@ -0,0 +1,49 @@
+# Example of a definition for a P5C060
+
+name = "P5C060"
+
+[pinout]
+# Defines both package structure and number of pins for each side. L, R, in order.
+pins_per_side = [12, 12]
+
+# List of all the pins of the IC, from 1 onward, as mapped on the ZIF42 socket of the dupico
+# note that, in this list, 21 is a placeholder for GND, 42 a placeholder for a power source (which could be 5V or different).
+# These are used to properly draw the map on the interface
+ZIF_map = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 21, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42]
+
+# List of the pins, as numbered on the IC, that can be used as clock inputs
+clk_pins = [1, 13]
+
+# List of the pins, as numbered on the IC, that are input only pins
+in_pins = [2, 11, 14, 23]
+
+# List of the pins, as numbered on the IC, that are I/O pins (can be configured as inputs or outputs)
+io_pins = [3, 4, 5, 6, 7, 8, 9, 10, 15, 16, 17, 18, 19, 20, 21, 22]
+
+# List of the pins, as numbered on the IC, that are output only
+o_pins = []
+
+# List of the pins, as numbered on the IC, that are registered outputs
+q_pins = [3, 4, 5, 6, 7, 8, 9, 10, 15, 16, 17, 18, 19, 20, 21, 22]
+
+# List of the pins that are active low OE
+oe_l_pins = []
+
+# List of the pins that are active high OE
+oe_h_pins = []
+
+# List of the pins, as numbered on the IC, that can be used as feedback pins
+f_pins = [3, 4, 5, 6, 7, 8, 9, 10, 15, 16, 17, 18, 19, 20, 21, 22]
+
+# List of the output pins, as numbered on the IC, capable of going HI-Z
+hiz_o_pins = [3, 4, 5, 6, 7, 8, 9, 10, 15, 16, 17, 18, 19, 20, 21, 22]
+
+[adapter]
+# pins that need to be forced high to read this in the adapter
+hi_pins = [24]
```

Comment:
Wondering about the pin 24 as high requirement.
This being a 24 pin device with, it should not require the TOML to specify any hi_pins to have power on pin 24, that is hardwired for PAL pod.
For 20 pin devices we specify pin 22 (of the ZIF42 socket, not of the ZIF24 in the pod) as high because it turns on the transistor that provides power to pin 22 of the ZIF24. 

Yep, the coincidence of both pins to be 22 is unintentional on my part...

Reply:
Ok i got that now. So in future should I leave that section with no pins declared?

Reply:
If the device uses pin 24 on the pal pod to draw power, yes, you can leave it empty. If it uses pin 22, putting pin 22 of the zif42 high will enable the transistor.

You will see further example of this once I build the pod to easily read 32/28/24 pin EPROMs. Those will have multiple transistors enabled by a combination of the hi_pins